### ISSUE TITLE
Customizing Options Page Slug "fw-settings" and/or other Option Page ideas

### ISSUE THREAD

**Issue by NoahjChampion** [ThemeFuse/Unyson#2597]
Hi. Let me know if I'm wrong about anything but I don't see options related to this and/or the rest described, which I would like to suggest if not already available. 

I found the filter hook `fw_backend_add_custom_settings_menu` which is also referenced @ https://github.com/ThemeFuse/Unyson/issues/31#issuecomment-59352722

But, how can I change the slug of "fw-settings" to a slug relative of my theme? 

I know I could change it in the core, but obviously seeing if there is a developer way first as I'm not sure what changing the fw-settings slug might affect. 

If there is not option for this I would like to request it. 

Also just to ask, if a them user using my theme uses a plugin using Unyson, won't there be an issue with the conflicting slug or the filter applied?

In terms of suggestions, my last framework had these options which were easy and relatively robust.

All options were relative to the instance itself, so these would be relative to the 
`fw()->theme->get_options($x),` file. 

```php
$settings = 
array(
	'option_name'    => '__settings__', // This set the database get_option()
	'menu_title'     => 'Theme Settings', // Menu Title
	'menu_type'      => 'menu', // menu, submenu, options, theme, etc. Type of menu
	'menu_position'  => '2', // Menu position
	'menu_slug'      => 'custom-theme-theme-settings', // Menu slug
	'menu_icon'      => 'dashicons-admin-site', // Menu icon
	'header'         => 'Theme Settings <small>by Fearless Mode Activated © 2016</small>', // Custom header text
	'sticky_header'  => TRUE, // Enable, disable sticky header
	'show_search'    => TRUE, // Enable disable search option
	'show_reset'     => TRUE, // Enable disable search option
	'show_reset_all' => TRUE, // enable disable reset settings option
	'show_footer'    => TRUE, // Enable disable the footer
	'footer'         => 'Theme <small> by Fearless Mode Activated</small>', // Custom footer text
	'ajax_save'      => TRUE, // Enable disable the ajax save option
);

$options = 'FW_Options_Array_Starts_Here'; 

// arrays.... 

Framework_Class::instance($settings, $options);
```

So in short a settings type operation on the options array file itself for option page arrays.
